src/strange_poker.c

```diff
@@ -71,6 +91,39 @@ int main(int argc, char *argv[])
             
             // show beginning of the game and wait for input
             ui->messageUser( ui->data, &ctx, "Start Game!" );
+
+            //preflop bids
+            uint64_t min_bid=20;
+            players[0].bid = 10;//small blind
+            players[1].bid = 20; //big blind
+            // need to switch the first player to the last position after the end of the round
+            int last_player=0;
+            uint32_t current_players = playerCnt;
+            for( uint32_t i=2; i < playerCnt; i++){
+                ctx.currentPlayer=i;
+                uint64_t pom = min_bid;
+                bidding( &min_bid, 4, players[i], &current_players, ui, &ctx);
+                if(min_bid > pom){
+                    if (i == 0) last_player = playerCnt-1;
+                    else last_player = i-1;
+                }
+            }
+            while(players[last_player].bid < min_bid){
+                for( uint32_t i=0; i < playerCnt; i++){
+                    ctx.currentPlayer=i;
+                    uint64_t pom = min_bid;
+                    bidding( &min_bid, 4, players[i], &current_players, ui, &ctx);
+                    if(min_bid > pom){
+                        if (i == 0) last_player = playerCnt-1;
+                        else last_player = i-1;
+                    }
+                }
+            }
+            ctx.moneyOnTable = min_bid * current_players;
```

Comment:
I do not think that is how bidding works. AFAIK everyone gives as much money as they declared or it goes around as long as not everyone agreed on the bet, while someone PASSES they leave the money on the table and loose all they possibly-winning rights.